packages/frontend/src/model/activate/dto/activate-request-body.dto.ts

```diff
@@ -1,5 +1,3 @@
-export interface ActivateRequestBodyDto {
-  identityEthAddress: string;
-  accountId: string;
-  signature: string;
-}
+import { components } from 'api-types';
+
+export type ActivateInputDto = components['schemas']['ActivateInputDto'];
```

Comment:
Filenames should be aligned with type names, in this case `activate-input.dto.ts`.